pr_agent/algo/ai_handlers/litellm_ai_handler.py

```diff
@@ -67,9 +76,9 @@ def __init__(self):
         if get_settings().get("HUGGINGFACE.API_BASE", None) and 'huggingface' in get_settings().config.model:
             litellm.api_base = get_settings().huggingface.api_base
             self.api_base = get_settings().huggingface.api_base
-        if get_settings().get("OLLAMA.API_BASE", None):
-            litellm.api_base = get_settings().ollama.api_base
-            self.api_base = get_settings().ollama.api_base
+        # if get_settings().get("OLLAMA.API_BASE", None):
```

Comment:
**Suggestion:** Consider adding a check for the existence of the `OLLAMA.API_BASE` setting before attempting to set the API base for Ollama. [possible issue, importance: 7]
```suggestion
        if get_settings().get("OLLAMA.API_BASE", None):
            if 'ollama' in get_settings().config.model:
                litellm.api_base = get_settings().ollama.api_base
                self.api_base = get_settings().ollama.api_base
```